rasa/cli/arguments/data.py

```diff
@@ -67,3 +68,56 @@ def set_validator_arguments(parser: argparse.ArgumentParser):
     )
     add_domain_param(parser)
     add_data_param(parser)
+
+
+def set_augment_arguments(parser: argparse.ArgumentParser):
+    add_config_param(parser)
+
+    parser.add_argument(
+        "--nlu-training-data", type=str, help="File containing your NLU training data."
+    )
+
+    parser.add_argument(
+        "--nlu-evaluation-data",
+        type=str,
+        help="File containing your NLU evaluation data.",
+    )
+
+    parser.add_argument(
+        "--nlu-classification-report",
+        type=str,
+        help="File containing your NLU classification report.",
```

Comment:
```suggestion
    parser.add_argument(
        "--nlu-classification-report",
        type=str,
        help="File containing your NLU classification report(e.g. `intent_report.json`). Generate it by training an NLU model on existing training data and then evaluating it with `rasa test nlu`.",
```

Reply:
Can we make this parameter optional and pick it up from `results` folder by default(if available)?